sklearn/model_selection/tests/test_search.py

```diff
@@ -623,25 +647,38 @@ def test_pandas_input():
     for InputFeatureType, TargetType in types:
         # X dataframe, y series
         X_df, y_ser = InputFeatureType(X), TargetType(y)
-        check_df = lambda x: isinstance(x, InputFeatureType)
-        check_series = lambda x: isinstance(x, TargetType)
+
+        def check_df(x):
+            return isinstance(x, InputFeatureType)
+
+        def check_series(x):
+            return isinstance(x, TargetType)
+
         clf = CheckingClassifier(check_X=check_df, check_y=check_series)
 
+        # Default scoring
         grid_search = GridSearchCV(clf, {'foo_param': [1, 2, 3]})
         grid_search.fit(X_df, y_ser).score(X_df, y_ser)
-        grid_search.predict(X_df)
         assert_true(hasattr(grid_search, "cv_results_"))
 
+        # Multi-metric scoring
+        for scoring in (('accuracy', ), ('accuracy', 'precision')):
+            grid_search = GridSearchCV(clf, {'foo_param': [1, 2, 3]},
+                                       scoring=scoring, refit='accuracy')
+            grid_search.fit(X_df, y_ser).score(X_df, y_ser)
+            assert_true(hasattr(grid_search, "cv_results_"))
+
 
 def test_unsupervised_grid_search():
     # test grid-search with unsupervised estimator
     X, y = make_blobs(random_state=0)
     km = KMeans(random_state=0)
-    grid_search = GridSearchCV(km, param_grid=dict(n_clusters=[2, 3, 4]),
-                               scoring='adjusted_rand_score')
-    grid_search.fit(X, y)
-    # ARI can find the right number :)
-    assert_equal(grid_search.best_params_["n_clusters"], 3)
+    for scoring in ('adjusted_rand_score', 'fowlkes_mallows_score'):
```

Comment:
It should've been `for refit in ('adjusted_rand_score', 'fowlkes_mallows_score'):`... I wanted to see if the multi-metric setup works cleanly for unsupervised...